clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v1/deploy/ops/servergroup/AbstractEnableDisableKubernetesAtomicOperation.groovy

```diff
@@ -66,6 +64,10 @@ abstract class AbstractEnableDisableKubernetesAtomicOperation implements AtomicO
 
     task.updateStatus basePhase, "Finding requisite server group..."
 
+    if (!supportsEnableDisable(credentials, description.serverGroupName, namespace)) {
```

Comment:
This should probably log that the operation has completed early because enable/disable is unsupported.